frappe/utils/__init__.py

```diff
@@ -75,11 +75,18 @@ def extract_email_id(email):
 		email_id = email_id.decode("utf-8", "ignore")
 	return email_id
 
-def validate_email_add(email_str, throw=False):
-	"""
-	validate_email_add will be renamed to the validate_email_address in v12
-	"""
-	return validate_email_address(email_str, throw=False)
+def validate_phone_number(phone_number, throw=False):
+	"""Returns True if valid phone number"""
+	if not phone_number:
+		return False
+
+	phone_number = phone_number.strip()
+	match = re.match("([0-9\ \+\_\-\,\.\*\#\(\)]){1,20}$", phone_number)
```

Comment:
```suggestion
	match = re.match("([0-9\ \+\_\-\,\.\*\#\(\)]){3,20}$", phone_number)
```

Alteast 3 characters for a valid phone number?

Reply:
There are some countries with 2 digit emergency numbers too. We'd rather just keep this flexible?

refs: https://en.wikipedia.org/wiki/List_of_emergency_telephone_numbers

Reply:
2 then?